src/items/ItemsDetailsPanel.component.tsx

```diff
@@ -147,6 +148,24 @@ function ItemsDetailsPanel(props: ItemsDetailsPanelProps) {
                   </Typography>
                 </Grid>
               </Grid>
+
+              <Grid item xs={12} sm={6} key={6}>
```

Comment:
```suggestion
              <Grid item xs={12} sm={6} key={7}>
```
This key needs changing to resolve the warnings in the logs: https://github.com/ral-facilities/inventory-management-system/actions/runs/8230847574/job/22505892697#step:6:132. There may be some in other files if that doesn't resolve all of them. I think maybe CatalogueItemsDetailsPanel is another.